docs/admin/eventing/broker-configuration.md

```diff
@@ -66,7 +67,7 @@ data:
       namespace: knative-eventing
 ```
 
-Now every Broker created in the cluster that does not have a `spec.config` will be configured to use the `kafka-channel` ConfigMap.
+The change in the example above would cause any new Broker that does specify a `spec.config` to inherit from the settings defined in the `kafka-channel` ConfigMap.
 
```

Comment:
I think use `default-br-config` throughout instead of `spec.config` - I went to the YAML looking for `spec:` and `config:` parts and that's not what's used in that configmap?

Reply:
spec.config is in the broker object, not a configmap